app/decorators/school_decorator.rb

```diff
@@ -6,4 +6,8 @@ def formatted_address
     address_parts.reject!(&:blank?)
     address_parts.join("<br/>").html_safe
   end
+
+  def formatted_inspection_date
+    last_inspection_date&.strftime("%e %B %Y")
```

Comment:
Is it worth moving this date format to the locales?

Reply:
Resolved in [Add date and time formats to localisation](https://github.com/DFE-Digital/itt-mentor-services/pull/81/commits/dd75a7cc585f05b1af9c9f8664934cc0162c802b)